packages/plugin-e2e/src/models/components/PanelEditOptionsGroup.ts

```diff
@@ -0,0 +1,30 @@
+import { Locator } from '@playwright/test';
+import { PluginTestCtx } from '../../types';
+import { Select } from './Select';
+
+export class PanelEditOptionsGroup {
+  constructor(private ctx: PluginTestCtx, public readonly locator: Locator, private groupLabel: string) {}
+
+  getRadioGroup(label = ''): Locator {
+    return this.getByLabel(label).getByRole('radiogroup');
+  }
+
+  async getSwitch(label = ''): Promise<Locator> {
+    // we need to add logic to select by a switch or a checkbox role depending on grafana version.
+    const id = await this.getByLabel(label).getByRole('checkbox').getAttribute('id');
+    return this.getByLabel(label).locator(`label[for='${id}']`);
+  }
+
+  getInput(label = ''): Locator {
+    return this.getByLabel(label).getByRole('textbox');
+  }
+
+  getSelect(label = ''): Select {
```

Comment:
I guess a panel option cannot exist without a label? So should probably not make this arg optional. Also, we may need to pass optional options in the future so we can for example let people override playwright options such as `exact`, `disabled` etc (see second arg of playwrights native `getByRole` or `getByLabel`.

Reply:
There are a couple of examples but I noticed that they occur when the label of the field is the same as the option group. Clock panel timezone is an example of this.